games/arcade/arcade_simulator.cpp

```diff
@@ -19,6 +19,7 @@
 #include "arcade.h"
 
 DECLARE_bool(pause_screen);
+DEFINE_string(ale_rom, "", "atari rom file");
```

Comment:
What if a default value "" is used for ale_rom? Will this cause any problem?

Reply:
ALE will report "could not open rom file" error.